cats/templates/cats/about.html

```diff
@@ -0,0 +1,11 @@
+<head>
+    <title>About</title>
+</head>
+{% extends 'base.html' %}
+{% block content %}
+    <h1>About</h1>
+    <div style="width: 500px; text-align: center; margin: auto">
+        <div>More projects:</div>
+        <a href="https://github.com/p-wel" target="_blank">https://github.com/p-wel</a>
```

Comment:
It is good idea to add `rel="noopener noreferrer"` while using `target="_blank"` to improve security https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Oh, thx
So easy to get someone phished, I have to try it 😂